cli/src/action/circuit/api.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use reqwest::{blocking::Client, header, StatusCode};
+use serde::Deserialize;
+
+use crate::error::CliError;
+
+/// A wrapper around the Splinter REST API.
+pub struct SplinterRestClient<'a> {
+    url: &'a str,
+}
+
+impl<'a> SplinterRestClient<'a> {
+    /// Constructs a new client for a Splinter node at the given URL.
+    pub fn new(url: &'a str) -> Self {
+        Self { url }
+    }
+
+    /// Fetches the node id of this client's Splinter node.
```

Comment:
"node ID" (per the Splinter capitalization guidelines at https://docs.google.com/document/d/1_-VwbB_YXdNMQJjcTIm5nOrhMECa9LF3lSgm7P_AhaU/edit#heading=h.ui1pysii1b8c )

Reply:
fixed